accelerator/config.c

```diff
@@ -66,10 +66,17 @@ status_t cli_core_set(ta_core_t* const core, int key, char* const value) {
         ta_log_error("Malformed input or illegal input character\n");
       }
       break;
-    case TA_THREAD_COUNT_CLI:
+    case HTTP_THREADS_CLI:
       strtol_temp = strtol(value, &strtol_p, 10);
       if (strtol_p != value && errno != ERANGE && strtol_temp >= 0 && strtol_temp <= UCHAR_MAX) {
-        ta_conf->thread_count = (uint8_t)strtol_temp;
+        ta_log_debug("Number of logical processors : %d, physical processors : %d\n", get_nprocs_conf(),
+                     get_nprocs_conf() / get_nthds_per_phys_proc());
+        if ((uint8_t)strtol_temp > MAX_HTTP_TPOOL_SIZE) {
+          ta_log_warning("Thread number %d exceed limitation %d. Abort option\n", (uint8_t)strtol_temp,
+                         MAX_HTTP_TPOOL_SIZE);
```

Comment:
Maybe add another line that assigns the `ta_conf->http_tpool_size` with `MAX_HTTP_TPOOL_SIZE`?

Reply:
Sounds good!

Reply:
Did you implement it?
I only saw the modified warning message.

Reply:
Sorry, I missed it.